### DISCUSSION TITLE
Lists of objects in a template within rich text body no longer working?

### DISCUSSION THREAD

**Discussion by atinsley67** [tinacms/tinacms#4033] [Help]
Hi! 

I have an object structure in my config file to allow my user to build a table in their blog posts. The table contains a list of row objects, where each row object contains a list of cell objects, and so on. You have to click on your table in edit mode, click the row you want to edit, click the cell etc, but it was pretty usable.

This was working in an earlier version of Tina, but today I upgraded to 1.5.11 and since then, when I click on a row object within the list of row objects in my widget, rather than showing me the row object edit screen, I get some sort of higher level Documents list in the right hand edit pane as follows:

![image](https://github.com/tinacms/tinacms/assets/19857073/0bd6841e-4894-4cfc-a24f-fc283b97288b)

This feels a like something is failing under the admin mode hood when trying to display the row object edit screen and is falling back to this higher level document screen.

This feels like a regression, but I'm also aware that I'm pushing the complexity of nested data structures here a bit. Is this something which is no longer supported, or perhaps an untested use case, or a known issue?

I'd be happy to share some config / code / screenshots if that would help.

Thanks,
Alex

**Comment by jeffsee55** [comment]
Yeah this sounds like a regression, if you can provide as much of your code as possible we can take a look. Thanks!

**Reply by atinsley67** [reply]
Hi Jeff,

Firstly, I think this is a duplicate of issue 3976 - https://github.com/tinacms/tinacms/issues/3976 . I'm getting a slightly different outcome (weird docs edit pane rather than blank edit pane) but the reproduction seems to be the same.

Here is my tina config file, which I link you to at risk of ridicule for some fairly hacky things you might see if you look around too carefully in my repo. The table template definition starts at line 213. You'll notice I have some (experimental) custom component and parse code in there, but I did try removing it and trying simpler reproductions (e.g. just a list of objects where each object contains only a string field) to get the same result. That's similar to the reproduction in 3976.

https://github.com/atinsley67/allaboutmomma/blob/main/.tina/config.tsx

FYI, I'm currently attempting to build a custom table editor component, which is working well enough, but it's a struggle to organize the data in the MDX file in a way that the Tina rich text edit mode will accept. I've fallen back on storing my table object data as escaped / stringified JSON, which is super ugly. I'd be interested to know whether this is something you (and your users) are coming across regularly - hitting complexity limits when trying to create more detailed components.

Thanks,
Alex
